source/_patterns/00-atoms/10-misc/07-image-responsive.mustache

```diff
@@ -0,0 +1,7 @@
+<picture>
+  <source srcset="{{ imgSrcResp-50.lg }}" media="(min-width: 60em)">
+  <source srcset="{{ imgSrcResp-50.md }}" media="(min-width: 48em)">
+  <source srcset="{{ imgSrcResp-50.sm }}" media="(min-width: 30em)">
+  <source srcset="{{ imgSrcResp-50.xs }}">
+  <img class="img-fluid" src="{{ imgSrcResp-50.fallback }}" alt="alternative text of the fallback image" width="600" height="300"/>
```

Comment:
I think **width** and **height** can be controlled with CSS, if we do it directly on an element, this will be read last from the browser, which can be confusing for the developer.